api/src/main/java/org/pmiops/workbench/notebooks/NotebooksServiceImpl.java

```diff
@@ -58,15 +70,17 @@ private ClusterRequest createFirecloudClusterRequest(String userEmail) {
             .combinedExtensions(ImmutableMap.<String, String>of())
             .labExtensions(ImmutableMap.<String, String>of()))
         .machineConfig(new MachineConfig()
-            .masterDiskSize(20 /* GB */)
-            .masterMachineType("n1-standard-2"));
+            .masterDiskSize(Optional.ofNullable(clusterOverride.masterDiskSize).orElse(20 /* GB */))
+            .masterMachineType(Optional.ofNullable(clusterOverride.machineType).orElse("n1-standard-2")));
   }
 
   @Override
-  public Cluster createCluster(String googleProject, String clusterName, String userEmail) {
```

Comment:
Correct - it was an inconsistency which was made even more apparent by this change so I fixed it. Namely - a cluster is always owned by the current credentials that called in, so it would never make sense for this to diverge from the request credentials.